packages/model-viewer/src/three-components/Renderer.ts

```diff
@@ -89,10 +101,13 @@ export class Renderer extends EventDispatcher {
     const webGlOptions = {
       alpha: true,
       antialias: true,
-      powerPreference: 'high-performance' as WebGLPowerPreference
+      powerPreference: 'low-power' as WebGLPowerPreference
```

Comment:
TODO: revert this before merging. This only makes it easier to test the dynamic resizing on a dual-GPU macbook.